Sources/Workspace/UserToolchain.swift

```diff
@@ -264,23 +270,22 @@ public final class UserToolchain: Toolchain {
             + destination.extraSwiftCFlags
     }
 
-    public init(destination: Destination, environment: [String: String] = ProcessEnv.vars) throws {
```

Comment:
I think we should retain the possibility to pass in a custom environment here.

Reply:
Thanks for pointing it out, I've reverted that part of the change.